Cargo.toml

```diff
@@ -10,3 +10,6 @@ members = [
   "neqo-interop",
   "test-fixture",
 ]
+
+[patch.crates-io]
+qlog = { git = "https://github.com/cloudflare/quiche", branch = "master" }
```

Comment:
I will remove the patch **before** merge once `qlog` `v0.10.1` is released. See also https://github.com/mozilla/neqo/pull/1505#discussion_r1421777643.